src/App.vue

```diff
@@ -424,6 +487,197 @@ export default {
     ]),
     ...mapActions("selectors", ["hideSelector"]),
     ...Utilities,
+    /**
+     * NOTE - In this method, "org" will be used in place of "workspace"
+     * to keep variable names short and readable.
+     *
+     * This method merges the user's and org's settings.
+     * This is needed to show both org level and user level settings in one settings menu.
+     * The global default settings object is used as a structure of the keys to iterate on.
+     * While iterating on the keys of the global default settings object, these rules are followed to merge
+     * - If a key is not present in org's settings, use the key from user's settings and skip to next key
+     * - If a key is present in org's settings, use the scope for that key and move to it's children
+     * - The above process is done for headers, tabs and atomic settings.
+     * - For the lowest level keys (atomic settings), use org's setting value if available otherwise use user's setting value
+     *
+     * @param {Object} userSettings - User's version of settings
+     * @param {Object} orgSettings - Org's version of settings
+     * @returns {Object} An object with user's and org's settings merged (with org's settings taking priority)
```

Comment:
```suggestion
     * @returns {Object} the settings to be rendered for the user
```

Reply:
But this is wrong, the settings to be rendered are not returned by this method. This method returns an object that is a merger of userSettings and workspaceSettings.

Reply:
use merge only and move out to utilities